packages/contracts/contracts/optimistic-ethereum/libraries/constants/Lib_PredeployAddresses.sol

```diff
@@ -5,14 +5,14 @@ pragma solidity >0.5.0 <0.8.0;
  * @title Lib_PredeployAddresses
  */
 library Lib_PredeployAddresses {
-    address public constant ovmL2ToL1MessagePasser = 0x4200000000000000000000000000000000000000;
-    address public constant ovmL1MessageSender = 0x4200000000000000000000000000000000000001;
-    address public constant ovmDeployerWhitelist = 0x4200000000000000000000000000000000000002;
-    address public constant ovmECDSAContractAccount = 0x4200000000000000000000000000000000000003;
-    address public constant ovmSequencerEntrypoint = 0x4200000000000000000000000000000000000005;
-    address public constant ovmETH = 0x4200000000000000000000000000000000000006;
-    address public constant ovmL2CrossDomainMessenger = 0x4200000000000000000000000000000000000007;
-    address public constant libAddressManager = 0x4200000000000000000000000000000000000008;
-    address public constant ovmProxyEOA = 0x4200000000000000000000000000000000000009;
-    address public constant ERC1820Registry = 0x1820a4B7618BdE71Dce8cdc73aAB6C95905faD24;
+    address constant L2_TO_L1_MESSAGE_PASSER = 0x4200000000000000000000000000000000000000;
```

Comment:
I removed the `public` keyword, as I was a bit confused about what it would actually do. I thought it might actually: 
1. add a getter 
2. result in a delegate call to read the value 

AFAICT from testing in remix neither happens, so it seems to have no effect.

Reply:
That's because the default visibility is `public` by default. However I _think_ we might save gas if we make them `private`.

Reply:
Isn't it nice to be able to `eth_call` to be able to get the values?

Reply:
I think if we're going to make this `public` then it's better to explicitly mark these as `public` rather than rely on the implicit conversion.

Reply:
> Isn't it nice to be able to `eth_call` to be able to get the values?
I don't think that's possible even prior to this PR being merged. 

1. It's poorly documented, but visibility keywords work quite differently in Libraries. Based on my testing, `public` doesn't provide a getter. This makes sense that we've only imported the file, not inherited it. 

2. If we want to enable something like 'ask the EM which predeploy address it uses for the DeployerWhitelist', then we'll have to explicitly add that getter.

Reply:
As a developer, I'd like to know what predeploys are deployed where. This could be achieved from importing `@eth-optimism/contracts` if the predeploys object is exported but that does not solve for all cases - ie using a different programming language. Should they go in the L2 address manager? It wouldn't be the best devex to have another place to `eth_call` that has the addresses and we risk them getting out of sync with 2 sources

Reply:
**Would it help to add the `internal` keyword?** 

I think the contention here is about making visibility explicit, but in the context of a Library, there is no real difference between `public`/`internal`/neither. Either way we need to explicitly define a getter if we want one (which IMO is much better, since we wouldn't want 8 or 9 extra getters everywhere this lib is imported).

Reply:
Tested locally for bytecode size of contracts that use the `Lib_PredeployAddresses` and there is no difference as to whether the visibility is specified or not or whether it's `public` or `internal` or even hardcoded in the contract. The compiler seems to treat them all the same.

Reply:
I added `internal`, which changes absolutely nothing but most closely matches the actual behavior, and people (rightfully) are looking for clarity on that point.